### PR TITLE
Change - Use Volley instead of VolleyPlus

### PR DESCRIPTION
**Pull request by seventhmoon**
As #99

### CODE REVIEW

**Comment by seventhmoon** [comment]
Travis failed to build because 'jdk: oraclejdk8' is missing from .travis.yml

**Comment by seventhmoon** [comment]
Done

File: skygear_example/build.gradle [hunk 1]
```diff
@@ -34,17 +35,23 @@ android {
...
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
             signingConfig signingConfigs.release
         }
     }
+
+    lintOptions {
+        abortOnError false
+    }
```
Comment (ben181231) [inline skygear_example/build.gradle]: After downgrading the build tools, this is no longer needed? 😅

Comment (seventhmoon) [inline skygear_example/build.gradle]: It is just default value, could also be removed

Comment (ben181231) [inline skygear_example/build.gradle]: 👍

**Comment by ben181231** [comment]
Picked as 5c8b257.

**Pull request closed without merging**